roles/openshift_prometheus/vars/main.yaml

```diff
@@ -1,2 +1,4 @@
 ---
 # vars file for openshift_prometheus
+openshift_prometheus_namespace: prometheus
```

Comment:
Is the intention to allow users to override these?  If so then they should be in main/default.yaml